### ISSUE TITLE
Can I do this

### ISSUE THREAD

**Issue by alejandroluengo** [philackm/ScrollableGraphView#156]
Hi there

I need to make the graph draw all points inside the view (no scroll needed)

Is that possible?

<img width="408" alt="captura de pantalla 2018-10-29 a las 12 06 58" src="https://user-images.githubusercontent.com/3824276/47646118-33b4a980-db73-11e8-97b4-eb916e0278b2.png">

Thanks for the help :)

**Comment by ivanglushko** [comment]
yeah just make 2 plots (LinePlot and DotPlot) with the same points
